DCC-CC4401/2023-1-Grupo-15
# Fix/login bug

## Summary
This pull request addresses an issue that was initially believed to be related to the login functionality. Upon further investigation, it was discovered that the actual problem lay in the user registration feature, which was not securely storing passwords in the database.

## Changes Made
1. Modified the SignUpView class to override the form_valid method, fixing the password storage issue in the registration process.
2. Improved the internal workflow of the login feature, enhancing data flow and task management.

## How to Test
- Register a new user via the registration page.
- Verify that the user can successfully authenticate using the password.
- Check that the stored password in the database is hashed.
